templates/submission/cancelled.tpl

```diff
@@ -0,0 +1,26 @@
+{**
+ * templates/submission/cancelled.tpl
+ *
+ * Copyright (c) 2014-2022 Simon Fraser University
+ * Copyright (c) 2003-2022 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * The page shown to the user when they cancel their submission.
+ *}
+{extends file="layouts/backend.tpl"}
+
+{block name="page"}
+	<h1 class="app__pageHeading app__pageHeading--center app__pageHeading--spacious">
```

Comment:
There's a mix of tabs and spaces here. I believe the `.tpl` files generally use spaces.

Reply:
fixed